Cargo.toml

```diff
@@ -406,3 +406,10 @@ tokio-postgres = { git = "https://github.com/madsim-rs/rust-postgres.git", rev =
 sqlx = { git = "https://github.com/madsim-rs/sqlx.git", rev = "3efe6d0065963db2a2b7f30dee69f647e28dec81" }
 # patch to remove preserve_order from serde_json
 bson = { git = "https://github.com/risingwavelabs/bson-rust", tag = "v2.14.0-json-no-preserve_order" }
+
+# patch to make zstd version compatible:
+# desired v4.9.0+2.10.0 requires: zstd-sys >= 2.0.15
+# conflect: parquet@54 requires: zstd-sys <= 2.0.13
+# why this patch work with zstd-sys < 2.0.15?
+# in the last release v4.8.0+2.3.0, the zstd-sys requirement is 2.0.6 < 2.0.13, and the release does not contain zstd related changes
```

Comment:
It's https://github.com/fede1024/rust-rdkafka/pull/771, which I don't quite get the idea of. Why did it bump everything while it didn't actually rely on the features from the latest minor version? IMO it only scattered the ecosystem.